### ISSUE TITLE
Schema directories containing index.ts causes issues in generated index.ts

### ISSUE THREAD

**Issue by dtymon** [mrjono1/joi-to-typescript#110]
When using `convertFromDirectory`, if the schema directory contains an `index.ts` file this results in the corresponding generated `index.ts` containing the TypeScript types defined directly inline rather than exporting via their generated files.

For example, without an `index.ts` in the schema directory, the generated `index.ts` correctly looks something like:
```typescript
export * from './Artist';
export * from './ArtistList';
export * from './NewArtist';
```
However, if an `index.ts` exists in the schema directory, the generated `index.ts` becomes:
```typescript
export interface Artist {
  ...
}
export interface ArtistList {
  ...
}
export interface NewArtist {
  ...
}
```
The generation of `index.ts` should be the same regardless of the presence of a schema `index.ts`

**Comment by vamcs** [comment]
We have this exact same problem and the generated `index.ts` file is gonna too big soon 😅. Isn't https://github.com/mrjono1/joi-to-typescript/pull/123 ready?

**Comment by mrjono1** [comment]
to be fixed by https://github.com/mrjono1/joi-to-typescript/pull/218

**Comment by mrjono1** [comment]
new option available in 4.0.0 to resolve this: `inputFileFilter`
